services/scabbard/libscabbard/src/store/error.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Error types and logic for CommitHashStores.
+
+use std::error::Error;
+use std::fmt::Display;
+
+use splinter::error::{
+    InternalError, InvalidArgumentError, InvalidStateError, ResourceTemporarilyUnavailableError,
+};
+
+/// Error type for the [CommitHashStore](super::CommitLogStore) trait.
```

Comment:
There are several places in the code and commit message that mentioned CommitLogStore, should they all be changed to CommitHashStore?

Reply:
Fixed